area.go

```diff
@@ -36,7 +45,7 @@ func (area *Area) Update(content string) {
 		}
 	} else {
 		for _, line := range lines {
-			fmt.Println(line)
+			fmt.Fprintln(area.writer, line)
```

Comment:
Due to the comments on https://github.com/atomicgo/cursor/pull/6 I assume this wouldn't work on Windows, so I have only added it to the non windows portion? Is this a correct assumption?